includes/class-freemius.php

```diff
@@ -227,6 +227,9 @@ class Freemius extends Freemius_Abstract {
 		 */
 		private static $_instances = array();
 
+		// Module types
+		const MODULE_TYPE_PLUGIN = 'plugin';
```

Comment:
@vovafeldman Sorry, please ignore my previous comment. Class constants are actually supported in 5.0: http://php.net/manual/en/migration51.oop.php#migration51.oop-constants.